kg_covid_19/transform_utils/ontology/ontology_transform.py

```diff
@@ -50,6 +51,10 @@ def parse(self, name: str, data_file: str, source: str) -> None:
         """
         print(f"Parsing {data_file}")
         transformer = ObographJsonTransformer()
-        transformer.parse(data_file, provided_by=source)
+        if data_file.endswith('.gz'):
+            compression = 'gz'
```

Comment:
mypy complains here, change this to:
`            compression: Optional[str] = 'gz'`